genesis/camino_unparsed_config_test.go

```diff
@@ -111,29 +83,44 @@ func TestParse(t *testing.T) {
 						Amount:            1,
 						NodeID:            sampleShortID.String(),
 						ValidatorDuration: 1,
-						DepositOfferID:    sampleID.String(),
 					}},
 				}},
 			},
 			err: fmt.Errorf("ID: %s is missing the prefix: %s", sampleShortID.String(), ids.NodeIDPrefix),
 		},
 		"Valid allocation": {
-			fields: fields{
+			unparsedCamino: UnparsedCamino{
 				VerifyNodeSignature: true,
 				LockModeBondDeposit: true,
 				InitialAdmin:        xAddress,
-				DepositOffers:       nil,
+				DepositOffers: []UnparsedDepositOffer{{
+					InterestRateNominator:   1,
+					StartOffset:             2,
+					EndOffset:               3,
+					MinAmount:               4,
+					MinDuration:             5,
+					MaxDuration:             6,
+					UnlockPeriodDuration:    7,
+					NoRewardsPeriodDuration: 8,
+					Memo:                    "offer memo",
+					Flags: UnparsedDepositOfferFlags{
+						Locked: true,
+					},
+				}},
 				Allocations: []UnparsedCaminoAllocation{{
 					ETHAddr:  "0x" + hex.EncodeToString(toShortID.Bytes()),
 					AVAXAddr: xAddress,
 					PlatformAllocations: []UnparsedPlatformAllocation{{
 						Amount:            1,
 						NodeID:            ids.NodeIDPrefix + sampleShortID.String(),
 						ValidatorDuration: 1,
-						DepositOfferID:    sampleID.String(),
+						DepositDuration:   1,
+						DepositOfferMemo:  "deposit offer memo",
```

Comment:
If we assume that allocation link DO by memo - just make it same as in DO above
```suggestion
						DepositOfferMemo:  "offer memo",
```

Reply:
That actually doesn't matter in this test, cause its only testing that unparsed structs are correctly parsed into parsed structs, but yea, I'll change it so it won't cause confusion.